flow/scripts/variables.yaml

```diff
@@ -652,6 +658,12 @@ CTS_CLUSTER_SIZE:
   default: 50
   stages:
     - cts
+CTS_REPAIR_TIMING_MAX_PASSES_PER_ITER:
+  description: >
+    Maximum number of repairs per iteration during repair_timing, called from CTS.
+  default: 20
+  stages:
+    - cts
```

Comment:
A proliferation of the same variable for the same operation. This multiplies the number of variables... Not ideal...